Format and specs changes

MiguelRomero90 opened the pull request:
### What does this Pull Request do?
- This pull request changes the format under JavaScript scripts to make it consitent.
- Modifies Specs file under '.feature' file to remove extra 'Then' usage.

### Why is this Pull Request needed?
This pull request is needed to standardize the code and make it consistent with our best practices.

### Are there any points in the code the reviewer needs to double check?
You may want to review the .feature file to make sure the changes in wording for the specs doesn't affect your behaviour.

### Are there any Pull Requests open in other repos which need to be merged with this?
This PR does not affect other pull requests.

### Addresses Issue(s):
This PR addressess the formatting problems for the scripts.

Ver 1.0.2

MikeRom0190 committed:
Format and specs changes

Changed format under JavaScript scripts to make it consitent.
Modified Specs file under '.feature' file to remove extra 'Then' usage.

c-sbonilla commented on cinepolis_proyecto/steps/cinepolisTest-steps.js:
```diff
@@ -4,96 +4,87 @@ var webdriver = require('selenium-webdriver');
+        
+        driver.findElement(By.xpath("/html//select[@id='cmbComplejos']")).click()
+        driver.sleep(500);
+        //driver.findElement(By.xpath("/html//select[@id='cmbComplejos']//option[5]")).click();  
```
Remove commented lines

MiguelRomero90 replied:
Comments removed under second commit

MikeRom0190 committed:
Removed comments

###What does this Pull Request do?

This pull request changes the format under JavaScript scripts to make it consitent.
Modifies Specs file under '.feature' file to remove extra 'Then' usage.

###Why is this Pull Request needed?

This pull request is needed to standardize the code and make it consistent with our best practices.

###Are there any points in the code the reviewer needs to double check?

You may want to review the .feature file to make sure the changes in wording for the specs doesn't affect your behaviour.

###Are there any Pull Requests open in other repos which need to be merged with this?

This PR does not affect other pull requests.

###Addresses Issue(s):

This PR addressess the formatting problems for the scripts.

Ver 1.0.2

c-sbonilla reviewed (approved):
LGTM
